expan/core/binning.py

```diff
@@ -17,14 +17,14 @@ class Bin(object):
     __hash__ = None
 
     def __init__(self, bin_type, *repr_args):
-        '''
+        """
```

Comment:
is it kind of misalignment between default quotes in different IDEs or was it a real mistake using single ticks?

Reply:
just for style consistency with other parts of code, nothing special.